### DISCUSSION TITLE
What is the Mapperly equivalent of AfterMap method in Automapper?

### DISCUSSION THREAD

**Discussion by ramarahul** [riok/mapperly#1421] [Q&A] [answered]
I'm trying to replace AutoMapper with Mapperly in our project due to the latter's performance. However, I'm facing difficulty while replacing AutoMapper with Mapperly because I can't find any equivalent of AfterMap() method of AutoMapper in Mapperly. What is the workaround available in Mapperly for this scenario?

**Comment by TimothyMakkison** [comment, marked as answer]
Hey, would [this](https://mapperly.riok.app/docs/configuration/before-after-map/) help?
